runtime/api/Robot.py

```diff
@@ -12,49 +12,494 @@
 >>> from api import Robot
 '''
 import memcache
-
 memcache_port = 12357
 mc = memcache.Client(['127.0.0.1:%d' % memcache_port]) # connect to memcache
 
 motor = {}
 
+def _lookup(name): #Returns actual UID given name
+    return name
+
 def get_motor(name):
     """Returns the current power value for a motor.
 
     :param name: A string that identifies the motor
     :returns: A value between -100 and 100, which is the power level of that motor.
```

Comment:
It seems that the API generally uses values in the 0.0-1.0 or -1.0 to 1.0 range, except where better units are available.

I suggest changing the motor API to use the same -1.0 to 1.0 range, for consistency